apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1918,9 +1918,13 @@ pay_close_mutual_fee(Fee, IAmt, RAmt) ->
        true                                   -> {0, RAmt - Fee + IAmt}
     end.
 
+-ifdef(TEST).
 close_mutual_defaults(_Account, _D) ->
-    #{ fee   => 20000}.
-
+    #{ fee   => 20000 }.
+-else.
+close_mutual_defaults(_Account, _D) ->
+    #{ fee   => 20000 * 1000000000 }.
```

Comment:
```suggestion
    #{ fee   => 20000 * 1000000 }.
```

Reply:
Why, I would just have to change it again in the miner config PR?

Reply:
I trust Ulf to solve this properly.